app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,23 @@
+class OrganisationsController < ApplicationController
+  def index; end
+
+  def edit
+    @organisation = Organisation.find_by(id: params.fetch(:id))
+  end
+
+  def update
+    organisation = Organisation.find_by(id: params.fetch(:id))
+    if organisation.update_attributes(organisartion_params)
```

Comment:
better to user `.update` than `.update_attributes`. See the difference here: https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html